RealmSwift/Tests/RealmTests.swift

```diff
@@ -249,6 +249,14 @@ class RealmTests: TestCase {
         XCTAssertEqual(try! Realm().objects(SwiftStringObject.self).count, 1)
     }
 
+    func testWriteReturning() {
+        let realm = try! Realm()
+        let object = try! realm.write {
+            return realm.create(SwiftStringObject.self, value: ["1"])
```

Comment:
[q] Nice, assumedly this gives us implicit returns as well?

Reply:
Yeah, implicit returns from single expression blocks are what results in unused return value warnings without `discardableResult`.